Client/Frontend/Strings.swift

```diff
@@ -676,6 +676,26 @@ extension String {
             tableName: nil,
             value: "Skip",
             comment: "String used to describes the option to skip the Sync sign in during onboarding for 106 version in Firefox Onboarding screens.")
+        public static let IntroNotificationTitle = MZLocalizedString(
+            "Onboarding.Notification.Title.v112",
+            tableName: nil,
```

Comment:
I think we should add tableName for new strings, in case we ever need them for messaging. Otherwise seems good to me!